.storybook/stories/datagrid/datagrid-detail.stories.ts

```diff
@@ -154,3 +156,12 @@ const DetailTemplate: StoryFn = args => {
 export const Detail: StoryObj = {
   render: DetailTemplate,
 };
+
+export const DetailOpened: StoryObj = {
```

Comment:
```suggestion
export const OpenDetail: StoryObj = {
```

for consistency with modal and side panel stories